apps/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -95,10 +95,24 @@ export class OperationExecutionManager {
       if (parallelism === 'max') {
         this._parallelism = numberOfCores;
       } else {
-        const parallelismInt: number = parseInt(parallelism, 10);
+        const parsed: number = parseInt(parallelism, 10);
 
-        if (isNaN(parallelismInt)) {
-          throw new Error(`Invalid parallelism value of '${parallelism}', expected a number or 'max'`);
+        if (isNaN(parsed)) {
+          throw new Error(
+            `Invalid parallelism value of '${parallelism}', expected a number, a percentage, or 'max'`
+          );
+        }
+
+        let parallelismInt: number = parsed > 0 ? parsed : 1;
+
+        if (
+          typeof parallelism === 'string' &&
+          parallelism.trim().endsWith('%') &&
+          parsed > 0 &&
+          parsed <= 100
```

Comment:
We should error if someone specifies something <0% or >100%.

Reply:
Added a check and error message for this.